Manifest.toml

```diff
@@ -137,15 +143,15 @@ version = "0.6.4"
 
 [[Formatting]]
 deps = ["Printf"]
-git-tree-sha1 = "aef8983c0f77c84ae77ced1bb3ecf5ab7d687700"
+git-tree-sha1 = "a0c901c29c0e7c763342751c0a94211d56c0de5c"
 uuid = "59287772-0a20-5a39-b81b-1366585eb4c0"
-version = "0.4.0"
+version = "0.4.1"
 
 [[GPUArrays]]
 deps = ["Adapt", "FFTW", "FillArrays", "LinearAlgebra", "Printf", "Random", "Serialization", "StaticArrays", "Test"]
-git-tree-sha1 = "b5009ac44b141ded5e6f04c4db83807970f56e91"
+git-tree-sha1 = "77e27264276fe97a7e7fb928bf8999a145abc018"
 uuid = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7"
-version = "1.0.2"
+version = "1.0.3"
```

Comment:
Why update `GPUArrays`?

Reply:
I did not do it consciously. I don't remember exactly but I had to manually add GPUArrays and it must have updated then. Can I just manually edit the Manifest.toml file to fix it?

Reply:
No, you should never manually edit `Manifest.toml`. It's ok, just wondering if it was related to the addition of `NCDatasets.jl`, somehow.